### PR TITLE
feat(getEmojiByCountry): Automatically generate flag emoji

### PR DESCRIPTION
**Pull request by Merkur39**
This PR improves flag emoji generation 🎉 

*Necessary development for ongoing development on MesPapiers*

### CODE REVIEW

File: packages/cozy-client/src/models/document/emojiCountry.js [hunk 1]
```diff
@@ -1,14 +1,26 @@
-const emojiCountry = {
-  fr: '🇫🇷'
-}
+import logger from '../../logger'
+
+// The offset between uppercase ASCII and regional indicator symbols
+const offset = 127397
```
Comment (JF-Cozy) [inline packages/cozy-client/src/models/document/emojiCountry.js:4]: comme c'est une constante on pourrait l'écrire un `OFFSET` et peut-être trouver un nom moins générique 🤔

**Review by JF-Cozy — APPROVED** [review]

File: packages/cozy-client/src/models/document/locales/index.js [hunk 2]
```diff
@@ -26,7 +26,14 @@ const getBoundT = lang => {
       ...opts,
       smart_count: opts?.smart_count || 1
     }
-    const emojiCountry = getEmojiByCountry(opts?.country, t)
+    const country = opts?.country
+
+    let emojiCountry
+    if (country === 'stranger') {
+      emojiCountry = t('country.stranger')
```
Comment (paultranvan) [inline packages/cozy-client/src/models/document/locales/index.js:33]: It looks weird to have a locale in the `emojiCountry`. Why note something like this?
```js
    const emojiCountry = country !== 'stranger' ? getEmojiByCountry(country) : null
    const countryLabel = country === 'stranger' ? t('country.stranger') : label

    return emojiCountry
      ? `${t(countryLabel, newOpts)} ${emojiCountry}`
      : t(countryLabel, newOpts)
```

File: packages/cozy-client/src/models/document/emojiCountry.js (hunk 1 above)
Comment (Merkur39) [reply packages/cozy-client/src/models/document/emojiCountry.js]: Ok, let's go to `UNICODE_OFFSET`.

File: packages/cozy-client/src/models/document/locales/index.js (hunk 2 above)
Comment (Merkur39) [reply packages/cozy-client/src/models/document/locales/index.js]: Indeed, it looks better 👍

**Review by paultranvan — APPROVED** [review]
Thanks!

File: packages/cozy-client/src/models/document/locales/index.js (hunk 2 above)
Comment (Merkur39) [reply packages/cozy-client/src/models/document/locales/index.js]: I've set it as below so as not to change the existing return.
```js
const emojiCountry = country !== 'stranger' ? getEmojiByCountry(country) : null
const strangerLabel = country === 'stranger' ? t('country.stranger') : null

return emojiCountry || strangerLabel
  ? `${t(label, newOpts)} ${emojiCountry || strangerLabel}`
  : t(label, newOpts)
```

**Pull request merged**